hawkular-accounts-feature-pack-resources/src/main/resources/subsystem-templates/hawkular-accounts-messaging-activemq.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<config default-supplement="default">
+  <extension-module>org.wildfly.extension.messaging-activemq</extension-module>
+  <subsystem xmlns="urn:jboss:domain:messaging-activemq:1.0">
+    <server name="default">
+            <?CLUSTERED?>
+
+      <security-setting name="#">
+        <role name="guest" send="true" consume="true" create-non-durable-queue="true" delete-non-durable-queue="true" />
+      </security-setting>
+
+            <?ADDRESS-SETTINGS?>
+
+      <http-connector name="http-connector" socket-binding="http" endpoint="http-acceptor" />
+      <http-connector name="http-connector-throughput" socket-binding="http" endpoint="http-acceptor-throughput">
+        <param name="batch-delay" value="50" />
+      </http-connector>
+      <in-vm-connector name="in-vm" server-id="0" />
+
+      <http-acceptor name="http-acceptor" http-listener="default" />
+      <http-acceptor name="http-acceptor-throughput" http-listener="default">
+        <param name="batch-delay" value="50" />
+        <param name="direct-deliver" value="false" />
+      </http-acceptor>
+      <in-vm-acceptor name="in-vm" server-id="0" />
+
+            <?BROADCAST-GROUPS?>
+            <?DISCOVERY-GROUPS?>
+            <?CLUSTER-CONNECTIONS?>
+
+      <jms-queue name="ExpiryQueue" entries="java:/jms/queue/ExpiryQueue" />
+      <jms-queue name="DLQ" entries="java:/jms/queue/DLQ" />
+      <jms-topic name="HawkularAccountsEvents" entries="java:/topic/HawkularAccountsEvents"/>
```

Comment:
I think we had a similar "problem" with the XML transformations. It's not necessarily a problem, just wondering if we need it all should we need to change a single line. It seems it's the case :)